src/sass/components/_galery-section.scss

```diff
@@ -0,0 +1,41 @@
+@import 'src/sass/utils/_vars.scss';
+
+.gallery {
+    background-image: linear-gradient(to top,#E5E5E5 80%,#FFF5F6 30%);
+    padding-bottom: 60px;   
+    @media screen and (min-width: 768px) {
+        padding-bottom: 60px; 
+    }
+ 
+    @media screen and (min-width: 1280px) {
+        padding-bottom: 120px; 
+    }
+}
+
+.gallery__grid {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 2.5px;
+    @media screen and (min-width: 768px) {
+        gap: 6px;
+    }
+    @media screen and (min-width: 1280px) {
+        gap: 8px;
+    }
+}
+
+.gallery__grid picture {
+    border-radius: 20px;
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+}
+
+.gallery__second-item {
```

Comment:
aici trebuie sa faci second chd si fourth chd, lasa o clasa globala gallery__item si pui comanda in fct de copil